net/discovery/src/main/java/de/danoeh/antennapod/net/discovery/PodcastIndexTrendingLoader.java

```diff
@@ -0,0 +1,66 @@
+package de.danoeh.antennapod.net.discovery;
+
+import de.danoeh.antennapod.model.feed.Feed;
+import de.danoeh.antennapod.net.common.AntennapodHttpClient;
+import okhttp3.CacheControl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.concurrent.TimeUnit;
+
+public abstract class PodcastIndexTrendingLoader {
+    private static final String API_URL = "https://api.podcastindex.org/api/1.0/podcasts/trending?max=%d&lang=%s";
+    private static final int NUM_LOADED = 25;
+
+    public static List<PodcastSearchResult> loadTrending(String country, int limit, List<Feed> subscribed)
+            throws JSONException, IOException {
+        String formattedUrl = String.format(Locale.ROOT, API_URL, NUM_LOADED, "de");
```

Comment:
Does this mean `de` will be used instead of the user's default language?

Reply:
This was just for testing. I needed to migrate the code from "region" (which we had previously) to "language" (which PodcastIndex uses)

The results are so much worse than the ones from iTunes :/ None of the results seems to actually be a top podcast here. The iTunes ones were pretty good.

Reply:
Podcast Index folks probably don't have enough data to build a top list. fingers corss that iTunes will fix their API?